docs/release-process.md

```diff
@@ -0,0 +1,184 @@
+
+## Known issues
+
+### Failure to update downstream crates
+
+There is a known issue involving repos such as ours which host multiple crates. Consider the dependency tree in our current configuration (`c2pa-crypto` -> `c2pa-rs` -> `c2patool`). It's possible for a crate that's earlier in the dependency cycle (e.g. `c2pa-crypto`) to have changes that warrant a new release and the crate that depends on it (`c2pa-rs` or `c2patool`) to have no commits since their prior releases.
+
+The current behavior of release-plz is to update the version reference in the downstream crate (`c2pa-rs` or `c2patool` in this example), but _not_ to generate a new release of the downstream crate.
+
+I've filed ([issue #2164](https://github.com/release-plz/release-plz/issues/2164)) to describe this issue. The developer of release-plz agrees that this is a bug.
+
+Colin has filed ([PR #2196](https://github.com/release-plz/release-plz/pull/2196)) to fix the issue, but that PR has not been merged as of this writing.
+
+In the meantime, the workaround that seems easiest is to post a no-op change to any downstream crates that need to be re-released. I typically find a whitespace change or a comment that could benefit from some proofreading. Those are enough to cause release-plz to call for a new release.
+
+### Left-behind release branches
+
+When performing the "Create a release pull request" step described above, release-plz sometimes updates the existing release branch and sometimes creates a new release branch with a new pull request. I don't understand how it makes that choice. When it creates a new pull request, it closes the old pull request, but does not delete the old release branch.
+
+In order to reduce the signal noise in the git repo, I've added a step in the release process to search for and delete old release branches. Search for "Clean up stale release-plz branches" in the [`release.yml`](../.github/workflows/release.yml) task.
+
+### `c2pa` crate accidentally published a 1.0.0 release
+
+When building and testing an earlier version of our release tooling (before adopting release-plz), we accidentally published a [version 1.0.0 of the `c2pa` crate](https://crates.io/crates/c2pa/1.0.0). We immediately yanked that version, but that means version 1.0.0 is permanently unavailable. When we do decide to release 1.0 of the `c2pa` crate, we will have to manually bump the version number to some other version (probably 1.0.1).
+
+## Troubleshooting
+
+### How to recover if the publish step fails or partially fails
+
+We have safeguards in place to ensure that a release PR will result in a successful publish, but they are not 100% effective. When there is a failure in the workflow, it's important to keep the core mental model in mind. (See "how it works" earlier in this document.) The following strategy _should_ work when release-plz fails to publish one or more crates from this project, though it may need some adaptation based on the specific failure mode:
```

Comment:
Could we split this section with sub-paragraph like:
Title: Observed symptom
Text: Steps to troubleshoot
(So it's easier to go to the right piece of content)

Reply:
In principle, I agree with this formation, but I'd like to hold off for now. Until we've observed multiple different failure modes and can have two or more entries of the form "Observed symptom," that structure feels like overkill.